### DISCUSSION TITLE
Your statement about Russia and Ukraine

### DISCUSSION THREAD

**Discussion by behrangsa** [avajs/ava#3072] [General]
Hi,

In your README file, you have added a banner that reads:

> Russia has invaded Ukraine and already killed tens of thousands of civilians, with many more raped or tortured [...]

May I ask why of all the wars and crimes happening around the world, only this one deserves your attention?

For example, Saudi Arabia has been in war with Yemen for many years now. Apparently a proxy war in Yemen with Iran.

> The country's humanitarian crisis is said to be among the worst in the world, due to widespread hunger, disease, and attacks on civilians. [1]

Or according the numerous credible reports, Uyghurs are being ethnically cleansed. [2]

Could you please explain why you show solidarity with Ukranians, but not with Yemenis or Uyghurs?

Sincerely,
Behrang

  [1] https://www.cfr.org/backgrounder/yemen-crisis#:~:text=The%20seven-year-old%20conflict,disease,%20and%20attacks%20on%20civilians.

  [2] https://www.bbc.com/news/world-asia-china-22278037

**Comment by sindresorhus** [comment]
A former member of the AVA team is directly impacted by the war in Ukraine and asked us to add the banner. We did so due to this personal connection.

We won't add banners for other causes, no matter how worthy our contributors or users may find them.

We won't facilitate a discussion around this either.
